service/static/js-src/muk.funder.js

```diff
@@ -3998,42 +3998,26 @@ $.extend(true, edges, {
                 var resultsId = edges.css_id(namespace, "results", this);
                 var results = "Loading...";
                 if (ts.terms.length > 0) {
-                    results = "";
-                    for (var i = 0; i < ts.terms.length; i++) {
-                        var val = ts.terms[i];
-                        if (val.count === 0 && this.hideEmpty) {
-                            continue
-                        }
-                        if ($.inArray(val.term.toString(), ts.selected) === -1) {
-                            results += '<div class="' + resultClass + '"><a href="#" class="' + valClass + '" data-key="' + edges.escapeHtml(val.term) + '">' + edges.escapeHtml(val.display) + "</a>";
-                            if (this.showCount) {
-                                results += ' <span class="' + countClass + '">(' + val.count + ")</span>"
-                            }
-                            results += "</div>"
-                        }
-                    }
-                }
-                var filterFrag = "";
-                if (ts.selected.length > 0) {
-                    for (var i = 0; i < ts.selected.length; i++) {
-                        var filt = ts.selected[i];
-                        var def = this._getFilterDef(filt);
-                        if (def) {
-                            filterFrag += '<div class="' + resultClass + '"><strong>' + edges.escapeHtml(def.display);
-                            if (this.showCount) {
-                                filterFrag += " (" + def.count + ")"
-                            }
-                            filterFrag += '&nbsp;<a href="#" class="' + filterRemoveClass + '" data-key="' + edges.escapeHtml(def.term) + '">';
-                            filterFrag += '<i class="glyphicon glyphicon-black glyphicon-remove"></i></a>';
-                            filterFrag += "</strong></a></div>"
+                    results = '<div class="form ' + resultClass + '">';
+                    for(i in ts.terms){
+                        val = ts.terms[i]
+                        if (val.count !==0 || !this.hideEmpty) {
+                            var sel = $.inArray(val.term.toString(), ts.selected)
+                            results += '<div class="form-fields__item-checkbox '
+                            results += sel !== -1 ? filterRemoveClass : valClass
+                            results += '" data-key="' + edges.escapeHtml(val.term) + '"><label><input type="checkbox"'+(sel !== -1 ? ' checked':'' )+'/> ' + edges.escapeHtml(val.display)
+                            results += this.showCount ? '<span class="' + countClass + '"> (' + val.count + ")</span>" : ''
+                            results += "</label></div>"
```

Comment:
Stylistically, I'd prefer a multi-line +, instead of concatenating to the var each time. 

Link only says adding to the DOM piecemeal is bad, and not how the string is put together in the first place.